### PR TITLE
Updates for Factorio Version 0.18

### PR DESCRIPTION
**Pull request by JDOGG88**
Changes:

1.  Had to remove the src folder as factorio would not recognize the mod with it. I know last time you said you like the file structure that way but I couldn't get it to work.

2. Updated the recipe to include the new higher tier pipes from bobplates.

3. Code tweaks to make the icons look right and for the mod to work in version 0.18.

### CODE REVIEW

**Comment by kendfrey** [comment]
> When you say it doesn't seem to recognize it, have you been copying the whole repository into the mods  folder, or just the contents of the src folder? I haven't tried it in 0.17 yet, but 0.16 needs the src folder only (renamed to the correct name).

_Originally posted by @kendfrey in https://github.com/kendfrey/high-pressure-pipes/pull/3#issuecomment-478273250_

I'm guessing you copied the whole repository. The mod only consists of the contents of the src folder.

**Comment by JDOGG88** [comment]
Yeah I did it again haha, pushed the reverted file structure.

File: src/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
+---------------------------------------------------------------------------------------------------
+Version: 1.0.4
+Date: 30. 03. 2020
+  Changes:
+    - Updated for Factorio Version 0.18.0
+    - Technology updated to new science pack recipe names.
```
Comment (kendfrey) [inline src/changelog.txt]: I don't see the change that this references. Am I missing something?

Comment (JDOGG88) [reply src/changelog.txt]: Oops the second line was supposed to be deleted.  Pushed it again haha

**Comment by kendfrey** [comment]
v1.0.4 is now published on the mod portal. Thanks once again!

**Pull request merged**